### PR TITLE
Fix go.mod for examples

### PR DESCRIPTION
**Pull request by srijan-27**
- Removed go.mod for all examples
- Moved migrations for pubsub to using pubsub examples from using-migrations
- Closes: #868

### CODE REVIEW

**Comment by Sianao** [comment]
The other example without a go.mod and go.sum

**Comment by srijan-27** [comment]
> The other example without a go.mod and go.sum

@Sianao - Yes, I am working to add go.mod for all examples.

**Comment by Sianao** [comment]
> > The other example without a go.mod and go.sum
> 
> @Sianao - Yes, I am working to add go.mod for all examples.

But if you add it , when new version release , we should update all example ?

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -208,4 +208,4 @@ jobs:
       - name: Check for file names errors
         uses: ls-lint/action@v2.2.3
         with:
-          config: .ls-lint.yml
\ No newline at end of file
+          config: .ls-lint.yml
```
Comment (Umang01-hash) [inline .github/workflows/go.yml:211]: Can we revert this change. Seems unwanted..

**Review by Umang01-hash — CHANGES_REQUESTED** [review]

**Review by Umang01-hash — APPROVED** [review]

**Review by vipul-rawat — APPROVED** [review]

**Pull request merged**